src/app/server-cluster/ServerClusterInterface.h

```diff
@@ -43,7 +43,12 @@ class ServerClusterInterface
     virtual ~ServerClusterInterface() = default;
 
     ///////////////////////////////////// Cluster Metadata Support //////////////////////////////////////////////////
-    [[nodiscard]] virtual ClusterId GetClusterId() const = 0;
+
+    /// The path to this cluster instance.
+    ///
+    /// This path (endpointid,clusterid) is expected to be fixed once the server
```

Comment:
nit, does `fix` mean permanent/constant, or changed once the server is used?

Reply:
remain constant/unchanged (i.e. it is assumed to be the primary lookup key and cannot change at runtime like). Updated the comment.